plugin/trino-resource-group-managers/src/main/resources/db/migration/mysql/V5__add_user_group_to_selectors.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE selectors ADD COLUMN user_group_regex VARCHAR(512);
```

Comment:
Why limit to 512? Is it because of MySQL?

Reply:
@hashhar just to be consistent with other existing columns. In our testing, we haven't hit this limit but I can see for groups you might want a bit more leeway. What do you think of 2048 instead?

Reply:
@aalbu any opinions?

I'd love to just be able to use unbounded types (since it's going to be a small table anyway) and not worry about this at all. 2048 sounds ok to me.

Reply:
@hashhar I updated it to 2048.

We could just make all string columns unbounded types. I can make a follow up PR to add migrations to do this.